pkg/cmd/cli/describe/describer.go

```diff
@@ -363,17 +369,66 @@ func (d *ImageDescriber) Describe(namespace, name string) (string, error) {
 		return "", err
 	}
 
-	return describeImage(image)
+	return describeImage(image, "")
 }
 
-func describeImage(image *imageapi.Image) (string, error) {
+func describeImage(image *imageapi.Image, imageName string) (string, error) {
 	return tabbedString(func(out *tabwriter.Writer) error {
 		formatMeta(out, image.ObjectMeta)
 		formatString(out, "Docker Image", image.DockerImageReference)
+		if len(imageName) > 0 {
+			formatString(out, "Image Name", imageName)
+		}
+		formatString(out, "Parent Image", image.DockerImageMetadata.Parent)
+		formatString(out, "Layer Size", units.HumanSize(float64(image.DockerImageMetadata.Size)))
```

Comment:
This is from the registry - why does upstream matter?  Is this committed in what is pushed to the registry?

> On May 19, 2015, at 2:34 PM, Andy Goldstein <notifications@github.com> wrote:
> 
> In pkg/cmd/cli/describe/describer.go:
> 
> >  	return tabbedString(func(out *tabwriter.Writer) error {
> >  		formatMeta(out, image.ObjectMeta)
> >  		formatString(out, "Docker Image", image.DockerImageReference)
> > +		if len(imageName) > 0 {
> > +			formatString(out, "Image Name", imageName)
> > +		}
> > +		formatString(out, "Parent Image", image.DockerImageMetadata.Parent)
> > +		formatString(out, "Layer Size", units.HumanSize(float64(image.DockerImageMetadata.Size)))
> Have to do an UPSTREAM for go-dockerclient to get it in
> 
> —
> Reply to this email directly or view it on GitHub.
>